Source/WebCore/platform/cocoa/PlatformSpeechSynthesizerCocoa.mm

```diff
@@ -90,9 +90,26 @@ - (WebSpeechSynthesisWrapper *)initWithSpeechSynthesizer:(WebCore::PlatformSpeec
         return nil;
 
     m_synthesizerObject = synthesizer;
+#if HAVE(AVSPEECHSYNTHESIS_VOICES_CHANGE_NOTIFICATION)
+    [[NSNotificationCenter defaultCenter] addObserver:self selector:@selector(availableVoicesDidChange:) name:AVSpeechSynthesisAvailableVoicesDidChangeNotification object:nil];
+#endif
     return self;
 }
 
+- (void)dealloc
+{
+#if HAVE(AVSPEECHSYNTHESIS_VOICES_CHANGE_NOTIFICATION)
+    [[NSNotificationCenter defaultCenter] removeObserver:self name:AVSpeechSynthesisAvailableVoicesDidChangeNotification object:nil];
```

Comment:
you shouldn't have to unregister anymore
https://useyourloaf.com/blog/unregistering-nsnotificationcenter-observers-in-ios-9/

Reply:
ah ok, will remove!